modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -3036,6 +3036,14 @@ cv::stereoRectify can be passed here. If the matrix is empty, the identity new c
 CV_EXPORTS_W void undistortPoints( InputArray src, OutputArray dst,
                                    InputArray cameraMatrix, InputArray distCoeffs,
                                    InputArray R = noArray(), InputArray P = noArray());
+/** @overload
+    @note Default version of cv::undistortPoints does 5 iterations to compute undistorted points.
+
+ */
+CV_EXPORTS_AS(undistortPointsExt) void undistortPoints( InputArray src, OutputArray dst,
```

Comment:
I would use suffix Iter instead of Ext. "Ext" is too generic and is not used in OpenCV. Can we make R & P non-optional parameters in this flavor and move the criteria to the end?